test/libsolidity/ASTJSON/unicode_legacy.json

```diff
@@ -131,7 +131,7 @@
                         "isPure": true,
                         "lValueRequested": false,
                         "token": "unicodeString",
-                        "type": "literal_string \"Hello \ud83d\ude03\"",
+                        "type": "literal_string hex\"48656c6c6f20f09f9883\"",
```

Comment:
I was wondering to actually return `literal_string unicode""`, `literal_string hex""`, or `literal_string ""` where appropriate, but not sure it is worth the effort.

Reply:
@chriseth and this ^^

Reply:
not worth the effort